manager/build.gradle.kts

```diff
@@ -89,6 +89,27 @@ fun Project.configureBaseExtension() {
             signingConfigs["debug"]
         }
 
+        val DEFAULT_APPLICATION_ID = "me.weishu.kernelsu"
+        val DEFAULT_APPLICATION_NAME = "KernelSU"
+
+        if (signFile.canRead()) {
+            val prop = Properties()
+            prop.load(signFile.inputStream())
+            var APPLICATION_ID = prop.getProperty("APPLICATION_ID")
```

Comment:
There is an overload method:

```
    public String getProperty(String key, String defaultValue) {
        String val = getProperty(key);
        return (val == null) ? defaultValue : val;
    }
```

So i think `prop.getProperty("APPLICATION_ID", DEFAULT_APPLICATION_ID)` is better.